Max resource info and wait note (#2888)
astronomer/docs: astro/deployment-settings.md

    - **Default Pod Size**:
        - **CPU**: The amount of CPUs that your tasks run with if no CPU usage is specified in their Pod configuration.
        - **Memory**: The amount of memory that your tasks run with if no memory usage is specified in their Pod configuration.
    
     For a Deployment running in a Hosted dedicated or shared cluster, the maximum possible **CPU** quota is 1600 vCPU and maximum **Memory** quota is 3200 GiB.
     

> 4. After you change the Pod size, wait for a couple of minutes before running your tasks. This is to allow the changes to be applied to the config map of your cluster. 

Reviewer: ```suggestion
4. After you change the Pod size, wait for a couple of minutes before running your tasks to allow Astro to apply the changes to your cluster config map. 
```

> 4. After you change the Pod size, wait for a couple of minutes before running your tasks. This is to allow the changes to be applied to the config map of your cluster. 

Reviewer: ```suggestion
4. After you change the Pod size, wait for a couple of minutes before running your tasks to allow Astro to apply the changes to your cluster config map. 
```
